docs/user/filesystem.md

```diff
@@ -259,25 +513,30 @@ $ daos container info --path <path to entry point>
 Please find below an example.
 
 ```bash
-$ dfuse -m /tmp/dfuse --pool tank --cont mycont
-$ cd /tmp/dfuse/
-$ ls
-foo
-$ daos cont create tank --label mycont3 --type POSIX --path ./link_to_externa_container
-  Container UUID : 933944a9-ddf2-491a-bdbf-4442f0437d56
+$ dfuse -m /scratch_fs/dfuse --pool tank --cont mycont
+$ cd /scratch_fs/dfuse/
+$ ls -l
+total 0
+-rw-rw-r-- 1 samirrav samirrav 0 Sep 23 16:31 foo
+$ daos cont create tank mycont3 --type POSIX --path ./link_to_externa_container
```

Comment:
"externa" or "external"?  Looks like you've missed a letter here and in the subsequent output.

In this example "mycont3" is the container label I assume?

Reply:
yes good catch. will fix it.